projects/igniteui-angular/src/lib/grids/cell.component.html

```diff
@@ -19,9 +19,9 @@
             formatter
                 ? (value | columnFormatter: formatter)
                 : column.dataType === 'number'
-                ? (value | igxdecimal: grid.locale)
+                ? (value | number: undefined:grid.locale)
                 : column.dataType === 'date'
-                ? (value | igxdate: grid.locale)
+                ? (value | date:'mediumDate':undefined:grid.locale)
```

Comment:
Don't pass `undefined` as timezone argument, just expose a new input for this in the grid column.